### PR TITLE
Redesign Past NUANSA overview

### PR DESCRIPTION
**Pull request by stevenwjy**
Move vertical navbar in past nuansa overview to the left and change the design

### CODE REVIEW

**Comment by stevenwjy** [comment]
waiting for license key (for open source) from fullpage.js

File: app/views/events/index.html.erb [hunk 1]
```diff
@@ -139,4 +139,4 @@
       <div style="clear: both;"></div>
   </div>
 </div>
-<%= javascript_include_tag('fullpage/past_nuansa.js') %>
+<%= javascript_include_tag('event-components/overview-fullpage.js') %>
```
Comment (chrisjwelly) [inline app/views/events/index.html.erb:165]: I would still like to move this line from the html file to the js file (separation of concerns), but when I do it, the page won't load with the fullpage effect. Would be great if someone could figure out why!

File: app/assets/javascripts/event-components/overview-fullpage.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  scrollOverflow: true,
+  navigation: true,
+  navigationPosition: 'left',
+  navigationBulletNames: [
+  	'2018', '2017', '2015', '2014', '2013', '2012', '2011', '2010', '2009', '2008'
+  ]
+});
+
+//methods
+window.fullpage_api.setAllowScrolling(true);
```
Comment (chrisjwelly) [inline app/assets/javascripts/event-components/overview-fullpage.js]: Newline at the end of the document
```suggestion
window.fullpage_api.setAllowScrolling(true);

```

File: app/assets/stylesheets/events.scss [hunk 3]
```diff
@@ -16,168 +16,114 @@ body {
...
+}
+
+/* Customise shape for active bullet */
+#fp-nav ul li a.active span, .fp-slidesNav ul li a.active span, 
+#fp-nav ul li:hover a.active span, .fp-slidesNav ul li:hover a.active span {
+  height: 70px;
+  width: 140px;
+  margin: 10px;
+  border-radius: 10px;
+
```
Comment (chrisjwelly) [inline app/assets/stylesheets/events.scss]: I think this is an extra empty line?
```suggestion
```

File: app/assets/stylesheets/events.scss [hunk 4]
```diff
@@ -16,168 +16,114 @@ body {
...
   }
 
-  .timeline ul li:nth-child(even):before {
-    visibility: hidden;
+  .content h4 {
+    margin-top: 150px;
   }
 
-  .timeline:before {
+
```
Comment (chrisjwelly) [inline app/assets/stylesheets/events.scss]: Another extra empty line?
```suggestion
```

File: app/views/events/index.html.erb [hunk 5]
```diff
@@ -17,9 +17,9 @@
 		<li> <%= link_to "Edit", edit_event_path(event) %> </li>
 	<% end %>
 <% end %>
-</div>
+</div>
```
Comment (chrisjwelly) [inline app/views/events/index.html.erb]: Very minor but there is a trailing whitespace perhaps added by accident
```suggestion
</div>
```

File: app/views/layouts/application.html.erb [hunk 6]
```diff
@@ -20,17 +20,16 @@
     <%= javascript_include_tag 'superfish/superfish.min.js', 'data-turbolinks-track': 'reload' %>
     <%= javascript_include_tag 'wow/wow.min.js', 'data-turbolinks-track': 'reload' %>
     <%= javascript_include_tag 'venobox/venobox.min.js', 'data-turbolinks-track': 'reload' %>
+
```
Comment (chrisjwelly) [inline app/views/layouts/application.html.erb]:
```suggestion
```

File: lib/assets/javascripts/fullpage/fullpage.js [hunk 7]
```diff
@@ -984,15 +988,16 @@
...
                 // Only add tooltip if needed (defined by user)
                 var tooltip = options.navigationTooltips[i];
-
+                
+                
                 if (typeof tooltip !== 'undefined' && tooltip !== '') {
                     li += '<div class="' + SECTION_NAV_TOOLTIP + ' fp-' + options.navigationPosition + '">' + tooltip + '</div>';
                 }
-
+
```
Comment (chrisjwelly) [inline lib/assets/javascripts/fullpage/fullpage.js]: Trailing whitespace
```suggestion

```

File: lib/assets/javascripts/fullpage/fullpage.js [hunk 8]
```diff
@@ -984,15 +988,16 @@
...
                 }
 
-                li += '<li><a href="#' + link + '"><span class="fp-sr-only">' + getBulletLinkName(i, 'Section') + '</span><span></span></a>';
+                li += '<li><a href="#' + link + '"><span class="fp-sr-only">' + getBulletLinkName(i, 'Section') + '</span></a>';
 
                 // Only add tooltip if needed (defined by user)
                 var tooltip = options.navigationTooltips[i];
-
+                
+
```
Comment (chrisjwelly) [inline lib/assets/javascripts/fullpage/fullpage.js]:
```suggestion
```

**Review by chrisjwelly — CHANGES_REQUESTED** [review]
I ran this branch on a local server and overall changes looks good to me so far!

I suggested several minor changes involving code style (mostly just removing additional whitespace or extra empty lines). Not urgent changes 👍

File: app/views/events/index.html.erb (hunk 1 above)
Comment (stevenwjy) [reply app/views/events/index.html.erb:165]: Yeah same, I also tried to move it but somehow it didn't work hahaha, can be added to the todo list first lol

File: app/views/events/index.html.erb (hunk 5 above)
Comment (stevenwjy) [reply app/views/events/index.html.erb]: :o how could you notice such thing lol

Comment (chrisjwelly) [reply app/views/events/index.html.erb]: Just somehow noticed it when going through the diff :) Github coloured the the extra whitespace nicely haha

**Review by chrisjwelly — APPROVED** [review]
LGTM :+1: 
Just need to wait for the license key

**Comment by stevenwjy** [comment]
Will be merged first and create another PR for the license key as it might take some time

**Pull request merged**